pkgs/applications/science/biology/MACS2/default.nix

```diff
@@ -2,32 +2,43 @@
 
 python3.pkgs.buildPythonPackage rec {
   pname = "MACS2";
```

Comment:
pnames are usually lowercase (SDL being the only exception).

Reply:
Thanks.
Renamed to `macs2` and added alias.